backend/src/mongo/models/Result.js

```diff
@@ -1 +1,16 @@
-// TODO: Result Model
+import mongoose from 'mongoose';
+import {restaurantSchema} from './Restaurant';
+
+const resultSchema = mongoose.Schema({
+  sessionID: {
+    type: String,
+    required: true,
+  },
+  restaurant: {
```

Comment:
Everything looks good. I would change restaurant to restaurants here though since it is an array of multiple restaurants